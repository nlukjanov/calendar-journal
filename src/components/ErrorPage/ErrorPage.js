import React from 'react';

const ErrorPage = () => {
  return <div>Something went wrong, please try again</div>;
};

export default ErrorPage;
