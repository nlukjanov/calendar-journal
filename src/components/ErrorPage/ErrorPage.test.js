import React from 'react';
// eslint-disable-next-line node/no-unpublished-import
import { render, waitFor } from '@testing-library/react';
import ErrorPage from './ErrorPage';

test('renders error message on error page', async () => {
  const { container, getByText } = render(<ErrorPage />);
  await waitFor(() =>
    expect(
      getByText(/Something went wrong, please try again/i),
    ).toBeInTheDocument(),
  );
  expect(container).toMatchSnapshot();
});
