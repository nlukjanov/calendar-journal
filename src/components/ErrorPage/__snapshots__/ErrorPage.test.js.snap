// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders error message on error page 1`] = `
<div>
  <div>
    Something went wrong, please try again
  </div>
</div>
`;
